src/Service.Host/client/src/components/requisitions/components/DepartmentNominal.js

```diff
@@ -107,15 +117,17 @@ DepartmentNominal.propTypes = {
     nominalCode: PropTypes.string,
     nominalDescription: PropTypes.string,
     setNominal: PropTypes.func.isRequired,
-    shouldRender: PropTypes.bool
+    shouldRender: PropTypes.bool,
+    enterNominal: PropTypes.bool
```

Comment:
Prop types are deprecated now - delete

Reply:
i can remove from here but if you search PropTypes in the solution they are on a number of other files probably ones that have originated in older solutions

Reply:
they're not from old solutions. i put them in as an easy way to keep the lint happy before it was upgraded even though i knew they were doomed